nilearn/input_data/nifti_masker.py

```diff
@@ -12,6 +12,7 @@
 from .. import _utils
 from .. import image
 from .. import masking
+from .. import plotting
```

Comment:
as @GaelVaroquaux if we really cannot avoid doing any plotting here this must be delayed to inside the reporting functions. most of nilearn can be used without matplotlib installed